src/richie/apps/courses/templates/courses/plugins/_organization_plugin.scss

```diff
@@ -0,0 +1,48 @@
+$richie-organization-plugin-organizations-margin-bottom: $grid-gutter-width;
```

Comment:
We need the `!default` flag here to make sure the values are overridable (http://sass-lang.com/documentation/file.SASS_REFERENCE.html#variable_defaults_default)


Also, I'm not sure why you repeat "organizations" in the variable names? Perhaps `$richie-organization-plugin-margin-bottom` is enough? And maybe `$richie-organization-plugin-title-fontsize` etc. for variables related to the title, like you did for the height?